index.html

```diff
@@ -59,9 +60,13 @@
               <i class="fas fa-atlas"></i>&nbsp;Teams</a>
           </li>
           <li class="nav-item">
-            <a href="../FAQ/index.html" class="nav-link p-3" style="font-size: large;">
+            <a href="FAQ/index.html" class="nav-link p-3" style="font-size: large;">
               <i class="fas fa-question-circle"></i>&nbsp;FAQ</a>
           </li>
+          <li class="nav-item">
+            <a href="Minecraft_BTE_Tips\index.html" class="nav-link p-3" style="font-size: large;">
```

Comment:
It should be `/` instead of `\`

Reply:
@Kira272921  just wondering can you tell if this is just copy pasted CSS from like a tutorial or another css sheet?

Reply:
I guess it is the copy paste of the CSS sheet of the main page

Reply:
> 
> 
> I guess it is the copy paste of the CSS sheet of the main page

hmm

Reply:
> It should be `/` instead of `\`

it works the same but for better convention, `/` is better. 

also specify it path so it be `./Minecraft_BTE_Tips/index.html` will make it safer for website deployer to read it

Reply:
Ok, @tintinkung @Kira272921 @StoneMcYT i apologise on the work i have given. im currently going to adjust the changes to your preferences. Just tell me what you think when i post it

Reply:
And btw, I just decided to remove the page entirely and replace the tips part with something totally different

Reply:
> And btw, I just decided to remove the page entirely and replace the tips part with something totally different

@S4ge-Rites could you make it to new branch? 
* 1 branch for landing page upgrade _(this branch)_
* 1 branch for your new page your are going to revamp from the tips page

Its easier to review like that not having 2 big project in the same pull request

Reply:
ok got it! thanks!